src/components/MegaMenu.astro

```diff
@@ -225,11 +225,11 @@ document.addEventListener("astro:beforeload", makeHeaderSticky);
   }
 
   .intersect .megamenu {
-    padding:4px 7px; 
-    border-radius: 24px; 
+    padding:4px 7px;
+    border-radius: 24px;
     /* outline: 1px solid rgba(255,255,255,0.18); */
     box-shadow: 0px 0px 2px var(--color-verse-500);
-    backdrop-filter: blur(24px);
+    backdrop-filter: brightness(0.5);
```

Comment:
let's try something like the following so that we don't lose the blur effect completely
backdrop-filter: brightness(.7) blur(24px)